resources.qmd

```diff
@@ -70,4 +70,26 @@ _Justice, Equity, Diversity, and Inclusion_
 -   [Office of Diversity and Inclusion](https://medschool.cuanschutz.edu/deans-office/diversity-inclusion)
 -   [DEI education, outreach, and recruitment](https://medschool.cuanschutz.edu/biochemistry/diversity-equity-inclusion)
 
+## Publication, Authorship & Research Misconduct
```

Comment:
@epbrenner if you want to add a copy of the links to `resources.qmd` and leave the descriptions in culture.qmd, that would be fine.